### ISSUE TITLE
[QUESTION] 使用定时任务问题

### ISSUE THREAD

**Issue by czetao** [WeiYe-Jing/datax-web#389]
![image](https://user-images.githubusercontent.com/43066081/100208072-444ad000-2f43-11eb-8b4a-5f132bc5bfdf.png)
使用定时任务启动任务,偶尔成功,偶尔失败,失败的没有运行日志,请问这是什么情况呢,
查看启动日志,没有发现输出日志.
![image](https://user-images.githubusercontent.com/43066081/100208226-72301480-2f43-11eb-9c1d-12d26fd8fdd1.png)

而且有个奇怪现象,服务部署在服务器,启动定时任务后,本地debug调试,会有接口请求进来....是我哪里部署有问题吗
[部署方式: 通过jar包的形式,将executor包和admin包分别部署在服务器]

**Comment by czetao** [comment]
请问你们的rpc框架是自己实现了一套吗,注册中心这方面有文档记录吗

**Comment by czetao** [comment]
![image](https://user-images.githubusercontent.com/43066081/100211212-d30d1c00-2f46-11eb-8657-2d32f49640e1.png)
很规律的,成功一次,失败3次

**Comment by LockiJiang** [comment]
1、本地有请求进来，那肯定是你本地的执行器注册到了服务器上
2、猜测执行器起了四个，其中三个服务器无法访问(比如上述提到的本地执行注册到了服务器上)，任务执行策略设置的轮询

**Comment by czetao** [comment]
非常感谢你的解答,请问下关于本地的执行器是如何注册的呢,是如何识别的

**Comment by LockiJiang** [comment]
官方文档有说明
**一、执行器管理**https://github.com/WeiYe-Jing/datax-web
![图片](https://user-images.githubusercontent.com/16129287/100303348-4494ab00-2fd7-11eb-844e-d95a2fad05f6.png)
**二、执行器注册配置**https://github.com/WeiYe-Jing/datax-web/blob/master/doc/datax-web/datax-web-deploy.md
![图片](https://user-images.githubusercontent.com/16129287/100303523-a523e800-2fd7-11eb-9479-33a41ac0d651.png)

**Comment by chaozhangpower** [comment]
配置任务信息截图，提供完整信息。请确认路由策略以及阻塞处理

**Comment by czetao** [comment]
![image](https://user-images.githubusercontent.com/43066081/100304049-e4066d80-2fd8-11eb-8d77-34fabd21ac64.png)
这是我的任务信息,我目前遇到的场景是,我在服务器上部署的项目 [admin+executor] ,然后在服务器上启动定时任务, 本地启动的admin模块,会有断点触发,路由策略是指 executor的回调吗 , 我部署在服务器上的回调是直接 127.0.0.1 
![image](https://user-images.githubusercontent.com/43066081/100304305-68f18700-2fd9-11eb-8ab6-5ea20abd4536.png)

**Comment by LockiJiang** [comment]
1、executor通过datax.job.admin.addresses向admin注册和回调admin
2、executor注册到admin后，admin保存executor地址，根据路由策略调用executor执行任务
也即是说，executor和admin会互相调用，关键在于executor中配置的datax.job.admin.addresses和datax.job.executor.appname
datax.job.admin.addresses为admin地址，datax.job.executor.appname为executor注册到admin的执行器名

**Comment by czetao** [comment]
是的,这部分我也是这样理解的,所以才很疑惑, 我datax.job.admin.addresses 是直接回调到服务器,127.0.0.1 ,怎么会回调到我本机启动的admin服务呢.

**Comment by LockiJiang** [comment]
如果是回调到了admin，那检查下本地是否有执行器启动，admin被调用只有两个地方，一是执行器注册，二是任务回调

**Comment by czetao** [comment]
好的,明白,我排查下.非常谢谢你们的解答

**Comment by waterWang** [comment]
> 好的,明白,我排查下.非常谢谢你们的解答

你好，问题是否得到解决？

**Comment by yunf17** [comment]
> ![image](https://user-images.githubusercontent.com/43066081/100211212-d30d1c00-2f46-11eb-8657-2d32f49640e1.png) 很规律的,成功一次,失败3次

我也遇到了同样的问题。cron表达式配置的是* 0/5 * ? * * *， 但任务管理页面的下次触发时间是以秒为单位的，而且是1秒

**Comment by Dadaniya** [comment]
目前发现是定时任务问题
